homeassistant/helpers/selector.py

```diff
@@ -71,6 +71,22 @@ class DeviceSelector(Selector):
     )
 
 
+@SELECTORS.register("area")
+class AreaSelector(Selector):
+    """Selector of a single area."""
+
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            # Area hass to contain entities provided by this integration
```

Comment:
Let's just keep it empty for now. That way we don't have to add frontend support right now.

I think that we have to think about what fields to include. I don't think `integration` is useful.

For showing in the UI, I also wouldn't want to filter out areas but instead show them as disabled or show a warning when selected that it doesn't match any of the required entities.

But for now let's keep it empty.